pkg/dns/service.go

```diff
@@ -60,12 +59,13 @@ func (s *Service) resolveIPS(ctx context.Context, addresses []gatewayv1beta1.Gat
 }
 
 func (s *Service) GetManagedHosts(ctx context.Context, traffic traffic.Interface) ([]v1alpha1.ManagedHost, error) {
-	managed := []v1alpha1.ManagedHost{}
+	var managed []v1alpha1.ManagedHost
 	for _, host := range traffic.GetHosts() {
-		managedZone, subDomain, err := s.GetManagedZoneForHost(ctx, host, traffic)
-		if err != nil {
-			return nil, err
-		}
+		//ToDo Should just continue on "no managed zone found for host" error
```

Comment:
is this todo still relevant?

Reply:
Fixing this at the moment, relates to the failing tests